runtime/include/j9cfg.h.in

```diff
@@ -251,6 +251,7 @@ extern "C" {
 #cmakedefine J9VM_OPT_USE_FFI_ONLY
 #cmakedefine J9VM_OPT_VALHALLA_VALUE_TYPES
 #cmakedefine J9VM_OPT_VALHALLA_FLATTENABLE_VALUE_TYPES
+#cmakedefine J9VM_OPT_VALHALLA_STRICT_FIELDS
```

Comment:
I thought one of the primary motivations for this new build flag was that it might become disassociated with Valhalla, in which case, `VALHALLA` should not appear in the name.

Reply:
It will still be associated with the Valhalla project but it may be released before JEP 401 which is the J9VM_OPT_VALHALLA_VALUE_TYPES flag that the other changes are under.

In that case J9VM_OPT_VALHALLA_STRICT_FIELDS would be replaced with `#if JAVA_SPEC_VERSION >=` for the Java version where its targeted for release.